packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -325,186 +477,56 @@ function useMediaUploadQueue() {
             currentTranscodingItem.current = id;
 
             if (isGifThatNeedsTranscoding) {
```

Comment:
I don't like the multiple elses here. Makes it a little hard to read. How about. 

```
 if (isTranscodingEnabled) {
            currentTranscodingItem.current = id;

            if (isGifThatNeedsTranscoding) {
              convertGifItem(item);
              return;
            } else if (canTranscodeFile(file)) {
              if (trimData) {
                trimVideoItem(item);
                return;
              } else if (muteVideo) {
                muteVideoItem(item);
                return;
              } else {
                // Transcode/Optimize videos before upload.
                // TODO: Only transcode & optimize video if needed (criteria TBD).
                // Probably need to use FFmpeg first to get more information (dimensions, fps, etc.)
                optimizeVideoItem(item);
                return;
              }
            } 
          
            uploadItem(item);
        }
```

Reply:
THat's just multiple else with returns in them? 🤔 

I suppose you mean just returning early and avoiding the elses?

Reply:
Fixed